README.md

```diff
@@ -9,6 +9,7 @@ This Data Migration project provides a flexible powerful engine based on the XRM
 ## Table Of Contents
 1. [Installation](#Installation)
 1. [Usage](#Usage)
+1. [Obfuscation](#Obfuscation)
```

Comment:
It seems like it would be better as a subheading under usage

Reply:
I'll make this change